test/features/deploy_kogito_build.feature

```diff
@@ -0,0 +1,70 @@
+# Disabled until CLI (used in PR check) supports KogitoRuntime and KogitoBuild - KOGITO-2064
+@disabled
+Feature: Deploy Kogito Build
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed
+
+  Scenario Outline: Build <runtime> remote S2I with native <native> using KogitoBuild
+    When Build <runtime> example service "<example-service>" with configuration:
+      | config | native | <native> |
+
+    Then Build "<example-service>-builder" is complete after <minutes> minutes
+    And Build "<example-service>" is complete after 5 minutes
+    And Deploy runtime <runtime> example service "<example-service>" with configuration:
+      | infinispan | useKogitoInfra | disabled |
+    And Kogito Runtime "<example-service>" has 1 pods running within 5 minutes
+    And Service "<example-service>" with process name "orders" is available within 2 minutes
+
+    @springboot
+    Examples:
+      | runtime    | example-service            | native   | minutes |
+      | springboot | process-springboot-example | disabled | 10      |
+
+    @smoke
+    @quarkus
+    Examples:
+      | runtime    | example-service         | native   | minutes |
+      | quarkus    | process-quarkus-example | disabled | 10      |
+
+    @quarkus
+    @native
+    Examples:
+      | runtime    | example-service         | native  | minutes |
+      | quarkus    | process-quarkus-example | enabled | 20      |
+
+#####
+
+  @smoke
+  @springboot
+  Scenario Outline: Build <runtime> binary build with native <native> using KogitoBuild
+    Given Clone Kogito examples into local directory
+    And Local example service "<example-service>" is built by Maven
+
+    When Build binary <runtime> service "<example-service>" with configuration:
+      | config | native | <native> |
+
+    Then BuildConfig "<example-service>" is created after 1 minutes
+    And Start build with name "<example-service>" from local example service path "<example-service>/target"
```

Comment:
I'm confused about this. Why do we need to start the build ourselves in the binary scenario? Using RemoteSourceBuildType is not needed. I would have expected to behave the same after running the Build binary ....

Reply:
Yes, we need to start the build by ourselves.
It is because we upload the binary using OC client, which is the separate step from KogitoBuild creation. Once we will have support for binary upload in CLI we can unify the steps (for CLI).

Reply:
Ok, can we add a comment here with a link to the ticket when this will be refactored?